### DISCUSSION TITLE
Discuss tree-shaking import feature

### DISCUSSION THREAD

**Discussion by justirva09** [ant-design/ant-design#46851] [General]
How does the 'tree-shaking' import feature work with Ant? I just want to reduce my build bundle size.

<img width="443" alt="image" src="https://github.com/ant-design/ant-design/assets/48151470/0319d2e0-5d6b-4982-952d-1d22c8cfad7a">

after see the image, I have question: How can I import the Ant Design (antd) library properly and efficiently to ensure smaller build results?

**Comment by zombieJ** [comment]
Seems your build config has not all from the es module which case the `lib` folder import.

**Reply by justirva09** [reply]
Could you please provide an explanation? I'm still curious about the "tree-shaking" method and its impact on the project. I've been reading about Ant Design using it, and I'm trying to initiate a discussion thread here. 

cc: @zombieJ

**Comment by FanetheDivine** [comment]
https://ant.design/docs/react/getting-started#import-on-demand
antd defaultly supports tree shaking
